modules/kubernetes-addons/crossplane/versions.tf

```diff
@@ -0,0 +1,10 @@
+terraform {
```

Comment:
Can this be moved to the kubernetes-addons module `versions.tf` ?

Reply:
It's throwing an error when i move this to `kubernetes-addons` . I think we need to keep this as is

```
╷
│ Error: Failed to query available provider packages
│ 
│ Could not retrieve the list of available versions for provider hashicorp/kubectl: provider registry registry.terraform.io does not have a
│ provider named registry.terraform.io/hashicorp/kubectl
│ 
│ Did you intend to use gavinbunney/kubectl? If so, you must specify that source address in each module which requires that provider. To see
│ which modules are currently depending on hashicorp/kubectl, run the following command:
│     terraform providers
╵

```

Reply:
Right, I faced the same error with `teams` feature before (same provider) and we left `versions.tf` too, just need to make sure consumers using the proper provider settings which you have in your example. Ty!